docs/source/install/systemrequirements.rst

```diff
@@ -0,0 +1,28 @@
+System Requirements
+===================
+
+|st2| requires 64-bit version of Ubuntu/Debian, RHEL, or CentOS. The table below lists the supported
+versions, along with Vagrant Boxes Vagrant Boxes and Amazon AWS instances we use for
+testing. Yes, using exactly the same boxes will improve your experience.
```

Comment:
Overall, I like it being in a separate file since this way we can also include it in other documents, if needed.